src/backend/error-message-to-user.ts

```diff
@@ -0,0 +1,3 @@
+export function errorMessageToUser(errorDetails: string) {
+  return `Es ist leider ein Fehler aufgetreten. Bitte wende dich an lars@serlo.org mit dieser Fehlermeldung: ${errorDetails}`
+}
```

Comment:
I guess for backend stuff we could (should) add sentry without privacy concerns.

Reply:
Yes! Some live notifications when we get a server side error/warning would be great to have long term. :relaxed:

Reply:
https://linear.app/serlo/issue/EDTR-313/setup-sentry-for-serlo-editor-as-lti-tool-backend